changelogs/changelog.yaml

```
@@ -119,6 +119,8 @@ releases:
     release_date: '2022-11-04'
   1.3.0:
     changes:
+      bugfixes:
+      - rabbitmq_publish - fix support for publishing headers as a part of a message (https://github.com/ansible-collections/community.rabbitmq/pull/174/)
```

```suggestion
      bugfixes:
      - rabbitmq_publish - fix support for publishing headers as a part of a message (https://github.com/ansible-collections/community.rabbitmq/pull/174/)
```
could you please remove this line and add a fragment https://docs.ansible.com/ansible/latest/community/development_process.html#creating-a-changelog-fragment ? This file is generated automatically during the release process and shouldn't be edited manually.

---

Should be resolved now, thanks